packages/clients/src/aptos/wallet.ts

```diff
@@ -447,7 +454,7 @@ export class ShinamiWalletSigner {
 
   /**
    * Sponsors, signs, and executes a MultiAgentTransaction gaslessly with the specified wallet as the sender.
-   * To call this method, your access key must be authorized for both the Aptos Wallet Service and Aptos Gas Station.
+   * To call this method, your access key must be authorized for both the target chain's wallet service and gas station.
```

Comment:
And, in the case of Aptos, Node Service: https://app.shortcut.com/shinami/story/5347/update-aptos-wallet-service-to-use-our-customer-s-node-service-key-rights-and-not-ours